lib/ssh_scan/api.rb

```diff
@@ -103,10 +110,25 @@ class API < Sinatra::Base
         options[:sockets] <<
           "#{params[:target]}:#{params[:port] ? params[:port] : "22"}"
         options[:policy_file] = options[:policy]
+        options[:force] = params[:force] ? params[:force] : false
+        unless options[:force] == 'true'
+          available_result = settings.db.fetch_available_result(params)
```

Comment:
I needed, params[:target] and params[:socket] both for finding the available result, so I used params instead.